qos/cometbft/endpoint.go

```diff
@@ -21,14 +24,20 @@ type endpoint struct {
 	// nil if no health check has been performed yet.
 	healthResponse *bool
 
-	// parsedBlockNumberResponse stores latest block height reported by the endpoint.
+	// chainIDResponse stores the chain ID of the endpoint.
+	chainIDResponse string
+
+	// syncedResponse stores if the endpoint is not catching up.
+	syncedResponse bool
+
+	// latestBlockHeightResponse stores latest block height reported by the endpoint.
 	// nil if no block height request has been made yet.
-	parsedBlockNumberResponse *uint64
+	latestBlockHeightResponse *uint64
```

Comment:
If this is based on the value of `sync_info. latest_block_height`, please call that out